cylc/flow/rundb.py

```diff
@@ -560,7 +560,7 @@ def execute_queued_items(self):
                 table.update_queues.clear()
             # Report public database retry recovery if necessary
             if self.n_tries:
-                LOG.warning(
```

Comment:
Not sure.

This isn't an issue for the public DB, but is for the private.

Reply:
I don't think this will get called if it's the private DB.